extract/extract/extractors/amazon_de.py

```diff
@@ -89,12 +89,12 @@ def extract_amazon_de(parsed_page: ParsedPage) -> Optional[Product]:
     if repairability_index := get_repairability_index(soup):
         sustainability_texts.append(repairability_index)
 
-    for energy_label in ["Energielabel", "Energy Label"]:
-        if energy_label in sustainability_texts:
-            if label_with_level := get_energy_label_level(soup):
-                sustainability_texts = [
-                    label.replace(energy_label, label_with_level) for label in sustainability_texts
-                ]
+    if label_with_level := get_energy_label_level(soup):
+        # check and remove Energy label strings that are extracted from CPF section
+        for energy_label in ["Energielabel", "Energy Label"]:
```

Comment:
This could be simplified if we treat both `energy_labels` and `sustainability_texts` as sets and do:
```
sustainability_texts = sustainability_texts.difference(energy_labels)
```
or 
```
sustainability_texts -= energy_labels
```